modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -136,10 +136,17 @@ internal class DiscoverAdapter(
     }
     var onChangeRegion: (() -> Unit)? = null
 
+    private var isDiscoverVisible = true
+
     private val imageRequestFactory = PocketCastsImageRequestFactory(context).smallSize().themed()
 
     init {
         setHasStableIds(true)
+        isDiscoverVisible = true
```

Comment:
There is no need to set it to `true` in `init` block since it is the default value of the property.